back-end/src/main/kotlin/Gneiss/PacketCompiler/DatabaseAccess/PacketDao.kt

```diff
@@ -31,4 +33,26 @@ class PacketDao(jsonSerializer: IJsonSerializer) : IPacketDao {
         }
         return ret
     }
+
+    override fun getAllKeys(): Set<String> {
```

Comment:
Could we make the return here a Set of Packets instead?

Reply:
We could, but I'm unsure if we want to for getting all packets. I saw this as returning the 'name' of all the packets which would be displayed on the front end. If we wanted to return a set of packets we could just use the 'name' field of the packet model to do the same job which seems fair. I'll give returning a set of packets a go, we can always change it back if it doesn't work well.